packages/zowe-explorer-api/CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ All notable changes to the "zowe-explorer-api" extension will be documented in t
 ### Bug fixes
 
 - Fixes an issue where properties of the `TableViewProvider` class were not accessible when the class was extended by developers. [#3456](https://github.com/zowe/zowe-explorer-vscode/pull/3456)
+- Fixed issue where the `AuthHandler.waitForUnlock` function could hang indefinitely if the profile is never unlocked. Now, the function will return after a 30 second timeout as a safety measure. This function should be used alongside the `AuthHandler.isProfileLocked` function to verify that the profile is unlocked before making API requests. [#3480](https://github.com/zowe/zowe-explorer-vscode/pull/3480)
```

Comment:
avoid future tense and rearrange the sentence a bit: "Now, as a safety measure, the function returns after a 30-second timeout ... ." 
or "As a safety measure, the function now returns after ... ."

Reply:
Sorry, I almost always check for future tense before uploading changelogs but did this in a rush.
Addressed in 94fec29